src/storage_broker/app.py

```diff
@@ -41,6 +40,39 @@ def handle_signal(signal, frame):
 signal.signal(signal.SIGTERM, handle_signal)
 
 
+def handle_legacy_validation(msg, decoded_msg, data, tracker_msg):
```

Comment:
I got to thinking about it and this validation path will actually hang around for a while as it's the only way currently to move things to the rejected bucket. Probably need to lose the legacy bit. The `announce` function is the one that will be going away once we get rid of the `available` topic